config/locales/nl.yml

```diff
@@ -49,9 +49,9 @@ nl:
     create:
       promise_resend:
     new:
-      submit:
-      title:
-      will_email_notice:
+      submit: Bevestiging opnieuw versturen
+      title: Bevestigingsemail opnieuw sturen
+      will_email_notice: We zullen een nieuwe email sturen met een bevestigings link
```

Comment:
`We zullen je een nieuwe email sturen met een bevestigingslink` (added `je` and `bevestigingslink` is one word)